This patch is based in part on a revert of parts of upstream commit
9feda988a51ec2bd97e83e0bbfd63af630059669

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -511,3 +511,3 @@
 [target."cfg(not(windows))".dependencies.termion]
-version = "4.0.0"
+version = ">= 1"
 optional = true
